src/c/profile/discovery/transport/udp_transport_datagram_posix_nopoll.c

```diff
@@ -0,0 +1,94 @@
+#include "udp_transport_datagram_internal.h"
+
+#include <sys/types.h>
+#include <sys/socket.h>
+#include <netdb.h>
+#include <unistd.h>
+#include <string.h>
+#include <errno.h>
+
+bool uxr_init_udp_transport_datagram(
+        uxrUDPPlatform* platform,
+        uxrIpProtocol ip_protocol,
+        const char* ip,
+        const char* port)
```

Comment:
I'm not able to find this function's signature. Shouldn't it be `bool uxr_init_udp_transport_datagram(uxrUDPTransportDatagram* transport)` instead?